src/packages/@environments/common/components/tabular-input/TabularInput.svelte

```diff
@@ -27,7 +27,7 @@
     if (keyValue) {
       pairs = [];
       pairs = keyValue;
-
+      let controller: boolean = false;
```

Comment:
why we have again declared this controller here, I think it is already declared above

Reply:
This is because it was needed to redeclare for the bug fix